omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -294,6 +294,28 @@
                         isValid = false;
                     }
                 }
+                                <!--Not to book on a public holiday-->
+               <!-- formatting date String from YYYY-MM-DD to DD-MM-YYYY to pass in the Url-->             
+               var selectedDate = jq('#appointmentDate').find("input[type=hidden]").val(); 
```

Comment:
@reagan-meant update this to use the date formatting for the number of clients on a fixed day

Reply:
Done